README.md

```diff
@@ -15,6 +15,8 @@ Installer `npm` si nécessaire, et exécuter la commande suivante.
 $ npm install && npm run dev
 ```
 
+Et se rendre sur http://localhost:7700
```

Comment:
Est-ce que l'URL n'est pas déjà donné sur la sortit standard, lorsqu'on lance la commande ? J'hésite à ajouter l'information ici "en double". Peut-être pourrions nous donner simplement une indication sur le fait que l'URL est donnée sur la sortie standard de la commande (cherchez la chaine "Project is running at"), sans préciser l'URL elle même ?

Reply:
Tu la retrouve effectivement dans la sortie, mais dans mon cas il fallait remonter dans le terminal et je n'ai pas trouvé cela super pratique.

Reply:
Oui, c'est pas super pratique c'est vrai, mais il s'agit de ne retrouver l'information qu'une seule fois. Ensuite tu la connais. 
Et si tu ne t'en rappelle pas plus tard, ce sera plus pratique de la retrouver dans la sortie standard que d'aller ré-ouvrir le readme.

Reply:
@etienneCharignon ce qui te gêne c'est d'alourdir le Readme ?
Sinon on pourrait écrire quelque chose de plus générique comme :

Et se rendre sur l'adresse affichée dans la sortie standard (http://localhost:7700 par défaut)

Reply:
Fait (en reprenant ton texte).